lib/components/login/bloc/login_block.dart

```diff
@@ -18,29 +18,37 @@ class LoginBloc extends Bloc<LoginEvents, LoginState> {
         yield* _signInEvent(states);
         break;
       case SignUpWithGoogleAuthEvent:
-        yield* _signInWithGoogleEvent(states);
+        yield* _signInWithGoogleEvent();
         break;
       case LogOutEvent:
-        yield* _singoutEvent(states);
+        yield* _singoutEvent();
+        break;
+      case CheckIfAccountExist:
+        yield* _checkIfAccountExistInDB((event as CheckIfAccountExist).uid);
         break;
       case SignUpEvent:
         break;
     }
   }
 
+  Stream<LoginState> _checkIfAccountExistInDB(String uid) async* {
+    var user = await isUserInCollection(uid);
+    yield LoggedIn(user);
+  }
+
   Stream<LoginState> _signInEvent(List<LoginState> states) async* {
     yield InProcessOfLogin();
     trySignInWithAsAnou();
     yield LoggedIn(null);
   }
 
-  Stream<LoginState> _singoutEvent(List<LoginState> states) async* {
+  Stream<LoginState> _singoutEvent() async* {
```

Comment:
At some point we also want to this refactor for the project and task blocks

Reply:
The method should be named _signOutEvent right?